src/deck-options.md

```diff
@@ -371,77 +371,70 @@ For more info about burying cards, please see [this section](./studying.md#sibli
+SuperMemo 2 (SM-2) scheduler. By more accurately determining when you are likely
 to forget, it can help you remember more material in the same amount of time.
-This setting is shared by all deck presets.
 
-When you enable the setting, some new options will
-become available, and SM-2 specific settings, such as "Graduating interval",
-"Easy bonus", etc, will be hidden.
+When you turn on FSRS, some new options
+become available, and SM-2 specific options, such as "Graduating interval",
+"Easy bonus", etc. is hidden. This option is shared by all deck presets.
 
 **Before Enabling**
 
 - Please ensure all of your Anki clients support FSRS. Anki 23.10, AnkiMobile 23.10,
-  and AnkiWeb all support it. AnkiDroid supports it in 2.17alpha3+. If
+  and AnkiWeb all support it. AnkiDroid supports it in 2.17+. If
   one of your clients doesn't support it, things will not work correctly.
 - If you previously used the 'custom scheduling' version of FSRS, please make
   sure you clear out the custom scheduling section before enabling FSRS.
 
-#### FSRS Options
-
-**Desired Retention**
+### Desired Retention
 
-Desired retention controls how likely you are to remember cards when they are reviewed.
-The default value of 0.9 will schedule cards so you have a 90% chance of remembering
-them when they come up for review again.
+Desired retention controls how likely you are to remember cards when they are scheduled for a review.
+The default value of 0.90 will schedule cards so you have a 90% chance of remembering
+them when they come up for review again. This should normally translate to remembering around 90% cards when they are reviewed, and only failing around 10%.
 
 Here is a graph that shows how adjusting this value will affect your workload:
 
 <img src="media/FSRS_retention.png" width="600">
 
 There are two things to notice:
 
-- As desired retention approaches 1.0, the frequency that you need to review cards
-  increases drastically. For example, imagine you have a card that you have a 90%
-  chance of remembering after 100 days. If your desired retention was 0.95, you'd
-  need to review it after 47 days instead (approximately twice as frequently).
-  At 0.97, the delay would be only 27 days (approximately 3.7x as frequently).
-  At 0.99, you'd be reviewing every 9 days (more than 10x what you'd be doing with
+- As desired retention approaches 1.0, the workload increases drastically.
+  Imagine you have a card with a 90% chance of remembering it after 100 days. If your desired retention is 0.90, you'll review the card again in 100 days. But if your desired retention is 0.95, you'll need to review it after 46 days instead.
+  This means the intervals of your cards almost halves at 0.95 desired retention and you need be reviewing cards twice as frequently compared to 0.90 desired retention.
+  At 0.97, the delay will be 27 days (you'll be reviewing 3.7x as frequently).
+  At 0.99, the delay will be only 9 days (more than 10x what you'd be doing with
```

Comment:
"At 0.99, the interval will be only 9 days (you'll have to review your cards more than 10x more frequently than with"